store/tooling/tooling_test.go

```diff
@@ -462,6 +463,16 @@ func (s *toolingSuite) Assertion(assertType *asserts.AssertionType, primaryKey [
 	return ref.Resolve(s.StoreSigning.Find)
 }
 
+func (s *toolingSuite) SeqFormingAssertion(assertType *asserts.AssertionType, sequenceKey []string, sequence int, user *auth.UserState) (asserts.Assertion, error) {
+	seq := &asserts.AtSequence{
```

Comment:
to avoid confusion this should probably panic for now if sequence is <= 0?

Reply:
Added a panic in that case